lib/absinthe/type/built_ins/scalars.ex

```diff
@@ -3,23 +3,43 @@ defmodule Absinthe.Type.BuiltIns.Scalars do
 
   @moduledoc false
 
-  scalar :integer, name: "Int" do
-    description """
-    The `Int` scalar type represents non-fractional signed whole numeric values.
-    Int can represent values between `-(2^53 - 1)` and `2^53 - 1` since it is
-    represented in JSON as double-precision floating point numbers specified
-    by [IEEE 754](http://en.wikipedia.org/wiki/IEEE_floating_point).
+  if Application.get_env(:absinthe, :use_legacy_non_compliant_integers, true) do
```

Comment:
not sure where in the documentation this should live or possibly the changelog so that way others are aware of this setting

Reply:
I have added it to the installation instructions and also information in the changelog. Do you think it is enough?